static/js/top_left_corner.js

```diff
@@ -54,13 +54,7 @@ function should_expand_pm_list(filter) {
     if (op_pm.length !== 1) {
         return false;
     }
-
-    const emails_strings = op_pm[0];
-    const emails = emails_strings.split(',');
-
-    const has_valid_emails = people.is_valid_bulk_emails_for_compose(emails);
-
-    return has_valid_emails;
+    return true;
```

Comment:
I think this isn't the right change -- won't this produce broken results with invalid (as opposed to deactivated) emails?  I think we want it to check the users are valid, just with a looser check that allows deactivated users.

Reply:
@timabbott thanks for the feedback. I've made some changes. Kindly review them.